ui/src/infra-model/view/form/infra-model-form.tsx

```diff
@@ -417,6 +400,39 @@ const InfraModelForm: React.FC<InframodelViewFormContainerProps> = ({
                             geometryPlan.author?.companyName
                         )}
                     </FormgroupField>
+
+                    <FormgroupField
+                        label={t('im-form.project-field')}
+                        qaId="project-im-field"
+                        inEditMode={fieldInEdit === 'project'}
+                        onEdit={() => setFieldInEdit('project')}
+                        onClose={() => setFieldInEdit(undefined)}>
+                        {fieldInEdit !== 'project' ? (
+                            project?.name
+                        ) : (
+                            <ProjectDropdown
+                                id={geometryPlan.project.id}
+                                setProject={(projectId) =>
+                                    changeInOverrideParametersField(projectId, 'projectId')
+                                }
+                                onAddProject={() => setShowNewProjectDialog(true)}
+                            />
+                        )}
+                    </FormgroupField>
+
+                    <FormgroupField
+                        label={t('im-form.file-field')}
+                        qaId="file-im-field"
+                        inEditMode={false}
+                        customAction={
+                            <InfraModelDownloadButton
+                                planId={geometryPlan.id}
+                                variant={ButtonVariant.GHOST}
+                                size={ButtonSize.SMALL}
+                            />
+                        }>
```

Comment:
Mietin olisikohan mahdollista muiluttaa tämä nappi jotenkin joko tuonne varsinaisen contentin sisälle tai sitten koko `FormgroupField`:in ulkopuolelle 🤔 Kokonaan uuden propsun tekeminen tätä varten kuulostaa hieman overkilliltä kun sillä on jo perussisältö `children`:in muodossa